setup-cloudfoundry/README.md

```diff
@@ -0,0 +1,19 @@
+# Setup Micro Bosh Deployment VM
+
+<a href="https://azuredeploy.net/" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template allows you to setup cloud foundry development envrioment. It will create a vm with public ipaddress,storage account, virutal network, network security group, 3 reserverd public ip 
```

Comment:
When is the documentation available? We need to either include more information in the README or link to the docs.